### PR TITLE
OSOE-128: Fix markdown files.

### PR DESCRIPTION
**Pull request by sarahelsaig**
OSOE-128

### CODE REVIEW

File: Lombiq.Tests.UI.AppExtensions/Readme.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # Lombiq UI Testing Toolbox - App Extensions
 
-
```
Comment (Piedone) [inline Lombiq.Tests.UI.AppExtensions/Readme.md:3]: For PRs opened to `dev` also open a `dev`-targeted OSOCE PR besides the `task/node-extensions` one.

File: Lombiq.Tests.UI/Docs/Troubleshooting.md [hunk 2]
```diff
@@ -32,24 +29,23 @@
...
 - If you can't get a link or button click working for some random reason but it's one that initiates a page leave then use `ClickReliablyOnUntilPageLeave()`.
-- When you're running a lot of tests in parallel then you may see random browser driver startup errors of the following sorts: 
-  > OpenQA.Selenium.WebDriverException: Creating the web driver failed with the message "Cannot start the driver service on http://localhost:50526/". Note that this can mean that there is a leftover web driver process that you have to kill manually.
+- When you're running a lot of tests in parallel then you may see random browser driver startup errors of the following sorts:
+  > OpenQA.Selenium.WebDriverException: Creating the web driver failed with the message "Cannot start the driver service on <http://localhost:50526/>". Note that this can mean that there is a leftover web driver process that you have to kill manually.
```
Comment (Piedone) [inline Lombiq.Tests.UI/Docs/Troubleshooting.md]: This doesn't need to be a link.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.Tests.UI/Docs/Troubleshooting.md (hunk 2 above)
Comment (DAud-IcI) [reply Lombiq.Tests.UI/Docs/Troubleshooting.md:33]: wth is this 😖

Comment (sarahelsaig) [reply Lombiq.Tests.UI/Docs/Troubleshooting.md]: It needs to be a link:
>  Troubleshooting.md(33, 125): [MD034] no-bare-urls: Bare URL used. An automatic fix is available with markdownlint-cli. Rule information: https://github.com/DavidAnson/markdownlint/blob/v0.26.0/doc/Rules.md#md034

Comment (0liver) [reply Lombiq.Tests.UI/Docs/Troubleshooting.md]: The docs on that are interesting!

Comment (Piedone) [reply Lombiq.Tests.UI/Docs/Troubleshooting.md]: This is a character-by-character quote of an exception message, and the localhost URL is not something that would make any sense to link to. So, this is a case where that rule shouldn't apply.

Comment (sarahelsaig) [reply Lombiq.Tests.UI/Docs/Troubleshooting.md]: ok, added magic string at the end of the line:

https://github.com/Lombiq/UI-Testing-Toolbox/blob/8634996433de2f10c49499ef648b29f83e9bbb33/Lombiq.Tests.UI/Docs/Troubleshooting.md?plain=1#L33

**Review by Piedone — APPROVED** [review]

**Pull request merged**